mne/report.py

```diff
@@ -1556,6 +1562,48 @@ def _render_forward(self, fwd_fname):
                                         repr=repr_fwd)
         return html
 
+    def _render_sensitivity_map(self, fwd_fname, subject, subjects_dir):
+        """Render Sensitivity plots.
+        """
+        caption = u'Sensitivity : %s' % fwd_fname
+        div_klass = 'sensitivity'
+        img_klass = 'Sensitivity'
+        show = True
+
+        fwd = read_forward_solution(fwd_fname, surf_ori=True)
+        mag_map = sensitivity_map(fwd, ch_type='grad', mode='fixed')
+        brain = mag_map.plot(time_label='Gradiometer sensitivity', subjects_dir=subjects_dir, clim=dict(lims=[0, 50, 100]))
```

Comment:
I suspect you are violating pep8 here (80 characters maximum for a line). You should be able to install a pep8 linter on your editor. If you use spyder, it already comes with one -- you just need to enable it. On sublime text, I think you should install the "anaconda" package